From f81b4e87a19de2ae55b6d2d74f427a11499f0012 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Sun, 20 Jun 2021 13:26:08 -0700
Subject: [PATCH 126/196] Update base seeds for static persistent nodes (#204)

Several people have setup persistent nodes online. This commit adds
them as we don't quite yet have DNS seeders up.
---
 contrib/seeds/nodes_main.txt | 4 ++++
 contrib/seeds/nodes_test.txt | 1 +
 src/chainparamsseeds.h       | 5 +++++
 3 files changed, 10 insertions(+)

diff --git a/contrib/seeds/nodes_main.txt b/contrib/seeds/nodes_main.txt
index e69de29bb..b27fee70b 100644
--- a/contrib/seeds/nodes_main.txt
+++ b/contrib/seeds/nodes_main.txt
@@ -0,0 +1,4 @@
+35.184.152.63:10605
+85.209.48.163:10605
+194.36.145.176:10605
+188.68.46.226:10605
diff --git a/contrib/seeds/nodes_test.txt b/contrib/seeds/nodes_test.txt
index e69de29bb..06c2c507d 100644
--- a/contrib/seeds/nodes_test.txt
+++ b/contrib/seeds/nodes_test.txt
@@ -0,0 +1 @@
+35.224.106.240:11605
diff --git a/src/chainparamsseeds.h b/src/chainparamsseeds.h
index 83d621683..0c06f270f 100644
--- a/src/chainparamsseeds.h
+++ b/src/chainparamsseeds.h
@@ -8,8 +8,13 @@
  * IPv4 as well as onion addresses are wrapped inside an IPv6 address accordingly.
  */
 static SeedSpec6 pnSeed6_main[] = {
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x23,0xb8,0x98,0x3f}, 10605},
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x55,0xd1,0x30,0xa3}, 10605},
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xc2,0x24,0x91,0xb0}, 10605},
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xbc,0x44,0x2e,0xe2}, 10605}
 };
 
 static SeedSpec6 pnSeed6_test[] = {
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x23,0xe0,0x6a,0xf0}, 11605}
 };
 #endif // BITCOIN_CHAINPARAMSSEEDS_H
-- 
2.39.5 (Apple Git-154)

